linux-pull-run.sh

```diff
@@ -89,8 +89,13 @@ if [[ ! $no_open ]]; then
 	fi
 fi
 
+# Create the Docker volume
+path=`pwd`
+mkdir -p recordedData
```

Comment:
This goes for `windows-pull-run.bat` as well: I would either prompt the user for the directory they want `recordedData/` to be added to, or I would add a note to the readme to tell the user to download/run the `linux-pull-run.sh`/`windows-pull-run.bat` file in the same directory that they want recorded data to be stored in.